projects/adp1050/src/examples/basic/basic_example.c

```diff
@@ -47,75 +57,123 @@ int basic_example_main()
+    printf(GREEN "PASSED!\n" RESET);
+	
 	ret = adp1050_unlock_pass(adp1050_desc, ADP1050_TRIM_DEFAULT_PASS,
 				  ADP1050_TRIM_PASS);
+	printf("UNLOCK TRIM: ");
 	if (ret)
-		goto exit;
-
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+	
 	ret = adp1050_set_close_loop(adp1050_desc);
+	printf("SET CLOSE LOOP: ");
 	if (ret)
-		goto exit;
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
 
 	ret = adp1050_vout_tr(adp1050_desc, ADP1050_VOUT_TR_200UV_US);
+	printf("VOUT TR: ");
 	if (ret)
-		goto exit;
-
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+	
 	ret = adp1050_vout_scale(adp1050_desc, -12, 341);
+	printf("VOUT SCALE: ");
 	if (ret)
-		goto exit;
-
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+	
 	ret = adp1050_vout_value(adp1050_desc, 0x3000, 0x5000);
+	printf("VOUT VALUE: ");
 	if (ret)
-		goto exit;
-
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+	
 	ret = adp1050_pwm_duty_cycle(adp1050_desc, 0x00DE, 0x000E, ADP1050_OUTA);
+	printf("PWM DUTY CYC: ");
 	if (ret)
-		goto exit;
-
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+	
 	ret = adp1050_set_pwm(adp1050_desc, ADP1050_OUTA, ADP1050_49KHZ);
+	printf("SET PWM: ");
 	if (ret)
-		goto exit;
-
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+	
 	ret = adp1050_read_value(adp1050_desc, &vout, &data, ADP1050_DUTY_CYCLE);
+	printf("READ VAL: ");
 	if (ret)
-		goto exit;
-
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+	
 	/* Checking statuses. */
 	ret = adp1050_read(adp1050_desc, ADP1050_STATUS_VOUT, &data, 1);
-	if (ret)
-		goto exit;
-
+	printf("READ STAT VOUT: ");
+		if (ret)
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+	
 	ret = adp1050_read(adp1050_desc, ADP1050_STATUS_INPUT, &data, 1);
+	printf("READ STAT INPUT: ");
 	if (ret)
-		goto exit;
-
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+	
 	ret = adp1050_read(adp1050_desc, ADP1050_STATUS_CML, &data, 1);
+	printf("READ STAT CML: ");
 	if (ret)
-		goto exit;
-
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+	
 	ret = adp1050_read(adp1050_desc, ADP1050_STATUS_WORD, data2, 2);
+	printf("READ STAT WORD: ");
 	if (ret)
-		goto exit;
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+	
+	ret = adp1051_llm_dllm_comm_setting(adp1050_desc, ADP1051_82US_7B, 
+							ADP1051_41US_6B_AVG, ADP1051_3LSB);
+	printf("READ LLM DLM COM SET: ");
+	if (ret)
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+
+	ret = adp1051_llm_dllm_setting(adp1050_desc, ADP1051_0MS, ADP1051_0mv, 1);
+	printf("READ LLM DLM SET: ");
+	if (ret)
+		printf(RED "FAIL!\n" RESET);
+    printf(GREEN "PASSED!\n" RESET);
+
+	adp1050_remove(adp1050_desc);
+
+	printf("===== END =====");
 
 exit:
 	if (ret)
 		pr_info("Error!\n");
 	adp1050_remove(adp1050_desc);
 	return ret;
-}
+}
```

Comment:
Ditto on empty line on EOF.
Also, If the existing basic_example also works on your device, you can leave it as it is.